strength_percentiles.py

```diff
@@ -4,89 +4,130 @@
-# program will be able to enter their own lift numbers and see where their
-# percentile rank among the competitive strength training population
+# This program scrapes revolutionpowerlifting.com for the results of a 2016
+# New York powerlifting meet, and loads them into a list of tuples.
+# Later versions of the program will allow users to enter their own lifting
+# numbers to be compared to the lifters from the competition.  They will be
+# able to see their percentile rank and where they would place among the
+# competitors in their demographic.
 
 import urllib2
 from bs4 import BeautifulSoup
 import re
 
-# Results for New York State Powerlifting meet
-
-quote_page = 'http://meets.revolutionpowerlifting.com/results/2016-meet-results/ny-states/'
-# contains list of tuples, each tuple (squat, bench, deadlift, total),
-# represents lifts made by individual of group corresponding to dictionary key
-lifts = {'all': [],
-        'amateur': [],
-        'professional': [],
+# The page from which the program collects powerlifting meet results
+QUOTE_PAGE = 'http://meets.revolutionpowerlifting.com/results/2016-meet-results/ny-states/'
+
+# Each row of the MEET_RESULTS_TABLE contains information corresponding to
+# one lifter in the competition.  Desired information is accessed
+# through the appropriate column of the table.  Column numbers start at 0.
+# Every other column contains a whitespace string.
+GENDER_COLUMN = 1 # Column containing gender information (male or female) for
+                  # all lifters
+PROFESSIONAL_STATUS_COLUMN = 3 # ...
+EQUIPMENT_COLUMN = 5 # ...
+SQUAT_COLUMN = 15 # ...
+BENCH_COLUMN = 17 # ...
+DEADLIFT_COLUMN = 19 # ...
+
+# This dictionary will contain lists of tuples.  Each tuple will contain one
+# lifter's (squat, bench, deadlift, total). The lists divide the results
+# by the lifters' category (ex: men, amateurs) but each lifter has
+# their result repeated in four lists, ("all," one of "men" and "women," one
+# of "amateur" and "professional," and one of "raw" and "equipped")
+categorized_results = {'all': [],
+        'amateurs': [],
+        'professionals': [],
         'men': [],
         'women': [],
         'raw': [],
         'equipped': []}
 
-page = urllib2.urlopen(quote_page)
-soup = BeautifulSoup(page, 'html.parser')
-
-# list of HTML table row tag objects, each corresponds to an individual's
-# performance at this meet
-meet_results_table = soup.find_all('tr')
-
-gender_column = 1
-professional_status_column = 3
-equipment_column = 5
-squat_column = 15 # column containing lifters' squat numbers, counting from 0
-bench_column = 17 # ...
-deadlift_column = 19 # ...
-
-for result in meet_results_table: # information for one lifter
-    contents = result.contents # list of HTML objects in row
-    if contents[1].get('colspan'): # row is column headers, no data, ignore
-        continue
+def scrape_webpage_and_store_meet_results(quote_page):
```

Comment:
Needs a docstring. Not a fan of this function name either, but am not quite sure what to rename it.